src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.cs

```diff
@@ -25,7 +24,19 @@ private static partial class Settings
         // This allows for the whole Settings nested class to be trimmed when Invariant=true, and allows for the Settings
         // static cctor (on Unix) to be preserved when Invariant=false.
         internal static bool Invariant => Settings.Invariant;
-#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS || TARGET_BROWSER
+#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
+        private static bool GetHybridGlobalizationSwitchValue() {
+            if (Settings.Invariant)
+                return false;
+            string? value;
+            TryGetStringValue("System.Globalization.Hybrid", "DOTNET_SYSTEM_GLOBALIZATION_HYBRID", out value);
+            if (value == null || value == "1")
+                return true;
+            return false;
```

Comment:
This is re-implementation of `AppContextConfigHelper.GetBooleanConfig`. You have `defaultValue` option there, that is used when `value == null`.
```suggestion
            return AppContextConfigHelper.GetBooleanConfig("System.Globalization.Hybrid", "DOTNET_SYSTEM_GLOBALIZATION_HYBRID", true);
```
or then we can even have a one - liner without a separate method, like the suggestion below

Reply:
Thanks, updated it.